crates/swc_css_minifier/src/compress/easing_function.rs

```diff
@@ -0,0 +1,145 @@
+use swc_css_ast::*;
+use swc_css_visit::{VisitMut, VisitMutWith};
+
+pub fn compress_easing_function() -> impl VisitMut {
+    CompressEasingFunction {}
+}
+
+struct CompressEasingFunction {}
+
+impl CompressEasingFunction {}
+
+impl VisitMut for CompressEasingFunction {
+    fn visit_mut_value(&mut self, value: &mut Value) {
+        value.visit_mut_children_with(self);
+
+        match value {
+            Value::Function(Function {
+                name,
+                value: function_value,
+                span,
+            }) if &*name.value.to_lowercase() == "cubic-bezier" && function_value.len() == 7 => {
+                match (
+                    &function_value[0],
+                    &function_value[2],
+                    &function_value[4],
+                    &function_value[6],
+                ) {
+                    (
+                        Value::Number(Number { value: first, .. }),
+                        Value::Number(Number { value: second, .. }),
+                        Value::Number(Number { value: third, .. }),
+                        Value::Number(Number { value: fourth, .. }),
+                    ) => {
+                        if *first == 0.0 && *second == 0.0 && *third == 1.0 && *fourth == 1.0 {
+                            *value = Value::Ident(Ident {
+                                span: *span,
+                                value: "linear".into(),
+                                raw: "linear".into(),
+                            })
+                        } else if *first == 0.25
+                            && *second == 0.1
+                            && *third == 0.25
+                            && *fourth == 1.0
+                        {
+                            *value = Value::Ident(Ident {
+                                span: *span,
+                                value: "easy".into(),
+                                raw: "easy".into(),
+                            })
+                        } else if *first == 0.42
```

Comment:
Is this correct?

Comparison of a float to.`0.0` and `1.0` will work, but I'm not sure if it's fine to compare with `0.42`.

Reply:
Nevermind, I checked original source code and it uses string to compare the value